src/features/focused_geometry_layer/renderers/FocusedGeometryRenderer.ts

```diff
@@ -1,5 +1,5 @@
 import { UserGeometryRenderer } from '~core/logical_layers/renderers/UserGeometryRenderer';
-import Icon from '../icons/marker_black.png';
```

Comment:
The `marker_black.png` file itself doesn't seem to be used anywhere after this change. Would be better to delete it

Reply:
I assume it is used in edit mode

Reply:
The black marker in edit mode comes from a different file (iconAtlas.png)

Reply:
Done